exercises/raindrops/test/test_raindrops.c

```diff
@@ -11,134 +12,106 @@ void tearDown(void)
 {
 }
 
+void convert_drops(int drops, char *expected)
+{
+   char result[16] = { 0 };
```

Comment:
Can then change this to `char result[BUFFER_LENGTH] = { 0 };`

Reply:
👍

Reply:
Might as well change buffer length to 16, since it cannot be any longer, might need those other 84 bytes someday.